frontend/src/types.ts

```diff
@@ -112,4 +112,30 @@ export interface LeaderboardEntry {
     yearlyWinRateShadow: number;
 }
 
+export interface ProcessedGameReport {
```

Comment:
This could probably be a mapped type to keep it in sync with `FormData`/`ValidFormData`/`GameReportPayload`, but for now just copied it straight from the backend